submissions/BukkyOyetimehin/HTML/product.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
```

Comment:
Kindly work on the existing product page you have. Make the necessary changes on that file